config/locales/sv.yml

```diff
@@ -570,6 +570,15 @@ sv:
       success: Användarens information uppdaterad.
       error: Ett problem hindrade användarinformationen från att uppdateras.
 
+    search_form:
+      profile_email: 'Profile Email'
+      search: 'Search'
+      filters: 'Filters'
+
+    user:
+      user_has_never_signed_in: har aldrig loggat in.
+      'yes': *yes
+      'no': *no
```

Comment:
I saw a couple of problems with translations:

1. User's search results in no records found:

<img width="795" alt="screen shot 2017-09-16 at 9 20 45 am" src="https://user-images.githubusercontent.com/9968213/30512707-07f98d0c-9ac3-11e7-9fe4-85443a02de40.png">

---

<img width="762" alt="screen shot 2017-09-16 at 9 39 54 am" src="https://user-images.githubusercontent.com/9968213/30512708-0c1f4e6c-9ac3-11e7-8a8b-c6b7f2f212ef.png">

2. Need Swedish translations for the search form labels _and_ the table column names:

<img width="925" alt="screen shot 2017-09-16 at 9 28 22 am" src="https://user-images.githubusercontent.com/9968213/30512714-2a430fc8-9ac3-11e7-9d3e-688877e283c4.png">

Reply:
Just FYI ---
The `i18n-tasks` gem has handy features for checking to see if things are missing from locale files.  I find it helpful to use the `missing` command for it like this:

````
> i18n-tasks missing
````

Run that at a terminal prompt in the Rails.root of the project.  It will show you entries that exist in 1 locale file but not the other, and it also looks through the views to see if you've referenced an i18n tag that isn't in your locale files.  

Since we call this in the `/spec/i18n_spec.rb` test, the gem is in the Gemfile, so you should have it installed already.

And of course you can look at the help for it: `i18n-tasks --help`

Reply:
Thank you @weedySeaDragon, that command is very useful!